engine/userguide/storagedriver/overlayfs-driver.md

```diff
@@ -4,412 +4,470 @@ keywords: container, storage, driver, OverlayFS
 title: Docker and OverlayFS in practice
 ---
 
-OverlayFS is a modern *union filesystem* that is similar to AUFS. In comparison
- to AUFS, OverlayFS:
+OverlayFS is a modern *union filesystem* that is similar to AUFS, but faster
+and with a simpler implementation. Docker provides a storage driver for
+OverlayFS.
 
-* has a simpler design
-* has been in the mainline Linux kernel since version 3.18
-* is potentially faster
+This topic refers to the Linux kernel driver as `OverlayFS` and to the Docker
+storage driver as `overlay` or `overlay2`.
 
-As a result, OverlayFS is rapidly gaining popularity in the Docker community
-and is seen by many as a natural successor to AUFS. As promising as OverlayFS
-is, it is still relatively young. Therefore caution should be taken before
-using it in production Docker environments.
+> **Note**: If you use OverlayFS, use the `overlay2` driver rather than the
+> `overlay` driver, because it is more efficient in terms of inode utilization.
+> To use the new driver, you need version 4.0 or higher of the Linux kernel.
+>
+> For more information about differences between `overlay` vs `overlay2`, refer
+> to [Select a storage driver](selectadriver.md#overlay-vs-overlay2).
 
-Docker's `overlay` storage driver leverages several OverlayFS features to build
- and manage the on-disk structures of images and containers.
+## Prerequisites
 
-Since version 1.12, Docker also provides `overlay2` storage driver which is much
-more efficient than `overlay` in terms of inode utilization. The `overlay2`
-driver is only compatible with Linux kernel 4.0 and later.
+OverlayFS is supported if you meet the following prerequisites:
 
-For comparison between `overlay` vs `overlay2`, please also refer to [Select a
-storage driver](selectadriver.md#overlay-vs-overlay2).
+- **Docker CE only**. OverlayFS is not supported on Docker EE. See
```

Comment:
Thinking here as well about distinguishing "supported", and "qualifies for commercial support"